webapp/src/styles/_compass-icons.scss

```diff
@@ -17,44 +17,43 @@
 @media screen and (-webkit-min-device-pixel-ratio:0) {
   @font-face {
     font-family: 'compass-icons';
-    src: url('../fonts/compass-icons/compass-icons.svg?45182295#compass-icons') format('svg');
+    src: url('../font/compass-icons.svg?94622077#compass-icons') format('svg');
```

Comment:
This change looks like not needed, it should keep untouched like the other lines at the beginning of the file.

Reply:
@jespino Done.